handlers/zoom_api_v1/readme.md

```diff
@@ -18,11 +20,9 @@ Used for requests to the Google Cloud Apps REST API Client
   - PATCH
   - DELETE
 
-**Path**
-  The relative API path (to the `https://api.zoom.us/v2` info value) that will be called.
-  This value should begin with a forward slash `/`.  A list of [zoom](https://marketplace.zoom.us/docs/api-reference/zoom-api) API paths. 
-**Body**
-  The body content (JSON) that will be sent for POST, PUT, and PATCH requests.
+**Path** :The relative API path (to the `https://api.zoom.us/v2` info value) that will be called. This value should begin with a forward slash `/`.  A list of [zoom](https://marketplace.zoom.us/docs/api-reference/zoom-api) API paths. 
```

Comment:
You reference "info value" here, but it appears the **Path** parameter is always relative to 'https://api.zoom.us/v2'.